src/testhost.x86/testhost.x86.csproj

```diff
@@ -9,8 +9,9 @@
     <TargetFrameworks>netstandard1.4;netcoreapp1.0;net451</TargetFrameworks>
     <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">netstandard1.4;netcoreapp1.0</TargetFrameworks>
     <WarningsAsErrors>true</WarningsAsErrors>
-    <PlatformTarget>x86</PlatformTarget>
-    <OutputType Condition="'$(TargetFramework)' != 'netstandard1.4'">Exe</OutputType>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Prefer32Bit>true</Prefer32Bit>
+    <OutputType>Exe</OutputType>
```

Comment:
please add condition TargetFramework != NS1.4, as we NS1.4 binary for running UWP tests, & we can't have it as exe.

Reply:
➡️ netstandard1.4 and netcoreapp1.0 both produce a .dll as output under the current settings. The output type cannot be conditionally set without breaking LAA.